pkg/manager/manager_options_test.go

```diff
@@ -1,54 +0,0 @@
-package manager
-
-import (
-	. "github.com/onsi/ginkgo/v2"
-	. "github.com/onsi/gomega"
-
-	"sigs.k8s.io/controller-runtime/pkg/config"
-
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/runtime"
-	configv1alpha1 "sigs.k8s.io/controller-runtime/pkg/config/v1alpha1"
-)
-
-var _ = Describe("manager.Options", func() {
```

Comment:
Shouldn't we keep the tests while the code is still there and only deprecated?
(similar in other cases)